packages/react-app-rewire-eslint/index.js

```diff
@@ -1,7 +1,10 @@
 function rewireEslint(config, env) {
-  delete config.eslint.configFile;
-  config.eslint.useEslintrc = true;
-
+  // when `react-scripts` version < 1.0.0, use next two line
+  // delete config.eslint.configFile;
+  // config.eslint.useEslintrc = true;
+  
+  // `react-scripts` >= 1.0.0
+  config.module.rules[0].use[0].options.useEslintrc = true;
```

Comment:
try to use getLoader helper from `react-app-rewired` so not to hard code loader position.
Also we can keep support older setup.
```
const eslintMatcher = (rule) => rule.loader === require.resolve('eslint-loader');
const eslintOptions = config.eslint || getLoader(config.module.rules, eslintMatcher);
eslintOptions.useEslintrc = true ;

```

Reply:
oh, it's really better... thx.